ddtrace/tracer.py

```diff
@@ -418,6 +421,7 @@ def configure(
 
         if sampler is not None:
             self._sampler = sampler
+            self._user_sampler = sampler
```

Comment:
Another sampler 😭. Is there any way we can consolidate some of our sampling configurations. It's becoming a pain to maintain.

Reply:
There's no way around it unfortunately. The user can define a custom sampler (through `tracer.configure(sampler=CustomSampler())` and we have to be able to reinstate it if they revert the Remote Configuration sampling rate.

We could choose to not revert back to a user sampler in this case since it's a rare use-case but it is possible given the public API today.